libnmstate/nm/route.py

```diff
@@ -210,3 +217,73 @@ def get_static_gateway_iface(family, iface_routes):
             if route[Route.DESTINATION] == destination:
                 return iface_name
     return None
+
+
+def get_routing_rule_config(acs_and_ip_profiles):
+    rules = []
+    for (_, ip_profile) in acs_and_ip_profiles:
+        for i in range(ip_profile.get_num_routing_rules()):
+            nm_rule = ip_profile.get_routing_rule(i)
+            rules.append(_nm_rule_to_info(nm_rule))
+
+    return rules
+
+
+def _nm_rule_to_info(nm_rule):
+    info = {
+        RouteRule.IP_FROM: _nm_rule_get_from(nm_rule),
+        RouteRule.IP_TO: _nm_rule_get_to(nm_rule),
+        RouteRule.PRIORITY: nm_rule.get_priority(),
+        RouteRule.ROUTE_TABLE: nm_rule.get_table(),
+    }
+    cleanup_keys = [key for key, val in info.items() if val is None]
```

Comment:
If you would have replace the `[]` with `()` it will make it a generator.
But this is a nit, do not bother about it.

Reply:
When using (), you will get error `RuntimeError: dictionary changed size during iteration`